packages/tx/src/capabilities/generic.ts

```diff
@@ -0,0 +1,85 @@
+import { SECP256K1_ORDER_DIV_2, bigIntToUnpaddedBytes, ecrecover } from '@ethereumjs/util'
+import { keccak256 } from 'ethereum-cryptography/keccak.js'
+
+import { Capability, type TypedTransaction } from '../types.js'
```

Comment:
Ah, I didn't know you could import a type inside a general import statement like this.  Neat!

Reply:
Found that out recently too!